benchmarks/plot_runs.py

```
@@ -0,0 +1,57 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def plot(benchmark):
```

Ruff (0.11.9)

Missing return type annotation for public function `plot`

Add return type annotation: `None`

(ANN201)